api/cloudprotocol/unitstatus.go

```diff
@@ -148,20 +149,28 @@ type InstanceStatus struct {
 
 // LayerStatus layer status.
 type LayerStatus struct {
-	LayerID   string     `json:"layerId"`
+	ID        string     `json:"id"`
 	Digest    string     `json:"digest"`
 	Version   string     `json:"version"`
 	Status    string     `json:"status"`
 	ErrorInfo *ErrorInfo `json:"errorInfo,omitempty"`
 }
 
+// ComponentItemStatus component status.
```

Comment:
Why?

Reply:
component may have several statuses on different nodes that's why we have list of statuses - ComponentItemStatus